largestPrimeFactor.js

```diff
@@ -1,6 +1,30 @@
 exports.largestPrimeFactor = function(n) {
   let primeNumber = 0;
-  // do your work here
+  let arrOfFactors = [];
+  let arrOfPrimes = [];
+
+  for (let i = 2; i < n; i++) {
```

Comment:
Can you think of how to merge this for loop with the one below so that you only have to go through the factors once?

Reply:
combined conditionals into one for loop, if statement, ready for review.